sys/dev/iwm/if_iwm.c

```diff
@@ -2602,6 +2599,7 @@ iwm_pcie_load_given_ucode_8000(struct iwm_softc *sc,
 	IWM_DPRINTF(sc, IWM_DEBUG_RESET, "working with %s CPU\n",
 		    image->is_dual_cpus ? "Dual" : "Single");
 
+	iwm_write_prph(sc, IWM_WFPM_GP2, 0x01010101);
```

Comment:
What's the reason for this?

Reply:
I think it's @wjguo just try to port from iwlwifi.

Reply:
it refers to https://elixir.bootlin.com/linux/latest/source/drivers/net/wireless/intel/iwlwifi/pcie/trans.c#L1075